security-analysis/security-analysis-api/src/main/java/com/powsybl/security/monitor/StateMonitor.java

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.security.monitor;
+
+import java.util.*;
+
+/**
+ * @author Etienne Lesot <etienne.lesot at rte-france.com>
+ *
+ *
+ * A stateMonitor allows to get information about branch and bus on the network after a security analysis computation.
+ * Contingency context allows to specify if the informations asked are about N situation N-k situation with a contingency id
```

Comment:
Small typo: `information` does not take an `s`.

Moreover, could you give an example of information it could get? How getting information work? I understand that StateMonitor keeps IDs of branches and voltage levels in memory, how does it work from that?